### ISSUE TITLE
pip version pin false positive

### ISSUE THREAD

**Issue by dimbleby** [hadolint/hadolint#287]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

A line like this:
    
    RUN pip install --index-url https://example.com/simple foobar==1.0.0

should be unproblematic.

### Actual behavior

    Dockerfile:3 DL3013 Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`

### Steps to reproduce the behavior

See above.

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.15.0-0-g23c3bda
```

Obviously similar to #67, but that looks as though it should have been fixed long ago.
